src/aof.c

```diff
@@ -1166,7 +1166,7 @@ void flushAppendOnlyFile(int force) {
         /* Successful write(2). If AOF was in error state, restore the
          * OK state and log the event. */
         if (server.aof_last_write_status == C_ERR) {
-            serverLog(LL_WARNING,
+            serverLog(LL_NOTICE,
```

Comment:
oran: from log message filtering perspective, this would be nice to have the same log level as the one with the failure, since they correspond to each other.
but on the other hand it doesn't indicate an error but rather recovery from error, so that from color highlighting perspective it should be NOTICE.
let's keep NOTICE.